### ISSUE TITLE
Add tlsCAFile support for MongoDB in Budibase Cloud

### ISSUE THREAD

**Issue by PvUtrix** [Budibase/budibase#10328]
**Describe the feature request**
I'm frustrated that I can't connect to my MongoDB cluster, which uses it's own root CA

Having an option to add a ca.pem file when adding a Mongo source would solve the problem.

**Comment by melohagan** [comment]
Hey @PvUtrix 

Are you self-hosting? Configuration was added to allow a certificate file to be specified: https://github.com/Budibase/budibase/pull/8260

**Comment by PvUtrix** [comment]
No, I was testing out the cloud version. This option is available on the self-hosted only?

**Comment by melohagan** [comment]
Hey @PvUtrix 

That's correct for now because the MongoDB client library expects a file, and does not allow you to provide a string key. 

I'll update the title to add support for TLS in cloud. This may involve an upload drop-zone to provide the cert file.
